scipy/special/cephes/chbevl.c

```diff
@@ -58,12 +58,9 @@
  */
 
 #include <stdio.h>
-#include "protos.h"
+#include "mconf.h"
 
-double chbevl(x, array, n)
-double x;
-double array[];
-int n;
+double chbevl(double x, double array[], int n)
```

Comment:
Just to clarify -- it looks like the old C code for the `chbevl` function had no types for its arguments? So, it wasn't being used / tested?

I see that the old header file did have types for the arguments in the function prototype, but I can't imagine there was ever a good reason to have this function defined without parameter types?

Reply:
Ohh.. I didn't know that! I see it is still supported in recent standards too.

Reply:
Arguably I shouldn't have changed this since it's tangential to the PR, but I couldn't resist. Unfortunately I'm still using bad Python C style since it should have been

```c
double
chbevl(double x, double array[], int n)
```

but we don't do that elsewhere in Cephes so ¯\\_(ツ)_/¯